ansible/vars.yml

```diff
@@ -5,3 +5,5 @@ srpm_dir: "{{ build_dir }}/srpms"
 rpms_dir: "{{ build_dir }}/rpms"
 mock_dir: "{{ build_dir }}/mock"
 git_repo_url: "https://git.samba.org/samba.git"
+ceph_base_url: false
+ceph_gpgkey: false
```

Comment:
Let's not make it a boolean. We expect a string, defined or not.

Reply:
@anoopcs9  wrote:
> Let's not make it a boolean.

I agree, and this confirms an earlier comment of mine.

>We expect a string, defined or not.

do you mean, ëmpty or not"? because empty string is defined,
 but could be considered undefined or not specified.

Reply:
Correct.